src/commands/debugger/logs/list.js

```diff
@@ -75,10 +76,25 @@ class DebuggerLogsList extends TwilioClientCommand {
 
   outputLogEvents(logEvents) {
     if (logEvents.length > 0) {
+      // Provide a human readable error message
+      logEvents = logEvents.map(e => {
+        return Object.assign({}, e, {
```

Comment:
Any reason to copy the event vs mutate it?

Reply:
Nope, changed it.